library/include/hipblaslt.h

```diff
@@ -79,7 +79,9 @@ typedef enum {
   HIPBLASLT_EPILOGUE_DGELU = 192,         /**<Apply gradient GELU transform. Requires additional aux input. */
   HIPBLASLT_EPILOGUE_DGELU_BGRAD = 208,   /**<Apply gradient GELU transform and bias gradient to the results. Requires additional aux input. */
   HIPBLASLT_EPILOGUE_BGRADA = 256,        /**<Apply bias gradient to A and output gemm result. */
-  HIPBLASLT_EPILOGUE_BGRADB = 512         /**<Apply bias gradient to B and output gemm result. */
+  HIPBLASLT_EPILOGUE_BGRADB = 512,        /**<Apply bias gradient to B and output gemm result. */
+  HIPBLASLT_EPILOGUE_SWISH = 1024,        /**<Apply Swish point-wise transform to the results (x:=Swish(x, 1)).*/
+  HIPBLASLT_EPILOGUE_SWISH_BIAS = 1028,   /**<Apply Bias and then Swish transform.*/
```

Comment:
Please shift the enum number to a larger one in case that has confliction with cublaslt enum in the future.
Besides, we use _EXT postfix for hipblaslt specific extension.
ex: HIPBLASLT_EPILOGUE_SWISH_EXT= 2^16
HIPBLASLT_EPILOGUE_SWISH_BIAS_EXT= 2^16 + 4

Reply:
Does this also apply to ROCBLASLT_EPILOGUE_SWISH?

Reply:
rocblaslt enum value should be the same with hipblaslt.
For the postfix _EXT, I don't think we need to add that in ROCBLASLT enum.

Reply:
https://github.com/ROCm/hipBLASLt/blob/0cb1d8dd70935e904bc16cc671104b1227982342/library/src/amd_detail/rocblaslt/src/utility.cpp#L211

It seems some other enums in rocblaslt do have _EXT?